source/_docs/mqtt/broker.markdown

```diff
@@ -22,13 +22,15 @@ Home Assistant contains an embedded MQTT broker. If no broker configuration is g
 | Port           | 1883 |
 | Protocol       | 3.1.1 |
 | User           | homeassistant |
-| Password       | Your API [password](/components/http/) |
+| Password       | no default value |
 | Websocket port | 8080 |
 
+> Since 0.77 release, embedded broker doesn't use your [API password](/components/http/) as default value. MQTT component will fail to setup if you have API passowrd configured, but don't have MQTT password confgiured likes following example.
```

Comment:
Couple of typo/spelling errors, and maybe add some "the"s for better reading flow.

*Since 0.77 release, the embedded broker doesn't use your [API password](/components/http/) as default password. The MQTT component will fail to setup if you have API password configured, but don't have MQTT password configured as in the following example.*